env/path/src/index.ts

```diff
@@ -9,18 +9,19 @@ export interface PrependDirsToPathResult {
 
 export function prependDirsToPath (prependDirs: string[], env = process.env): PrependDirsToPathResult {
   const prepend = prependDirs.join(path.delimiter)
-  if (env[PATH] != null && (env[PATH] === prepend || env[PATH]!.startsWith(`${prepend}${path.delimiter}`))) {
+  const envPath = env[PATH]
+  if (envPath != null && (envPath === prepend || envPath.startsWith(`${prepend}${path.delimiter}`))) {
     return {
       name: PATH,
-      value: env[PATH]!,
+      value: envPath,
       updated: false,
     }
   }
   return {
     name: PATH,
     value: [
       prepend,
-      ...(env[PATH] != null ? [env[PATH]] : []),
+      envPath ?? '',
```

Comment:
We can go directly to a string instead of creating an array and spreading it.

Reply:
Hm. This change does add `;` to the end if `envPath` is undefined. While that's probably fine, I'll revert this to keep existing behavior.